docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -227,6 +227,8 @@ h|Type
 h|Default value
 |`2`
 
+(For trial Kafka instances, the default value is `1`.)
+
 h|Kafka property name
 |`min.insync.replicas`
 |===
```

Comment:
Luke, not part of this PR, but have a look at the "Retention time" and "Retention size" below at lines 237 and 262. Note how they are of type `Long` but either the default value or the supported values or both show just the measurements supported as string values, or the value `Unlimited`. I didn't notice that before, but seems odd for a `Long` type to mention strings for supported or default values. 

How about we simply remove the "Supported values" row in the two tables and change the default value for Retention size from `Unlimited` to `9223372036854775807 ms` (what Kafka considers unlimited)? That work or is there something I've misunderstood?

Reply:
@showuon , are you sure about `-1`? The value that Kafka seems to use for "unlimited" for this property and for many others is `9223372036854775807`, which is thousands of years and virtually unlimited. To have -1 bytes doesn't sound unlimited, in fact it sounds like it would disallow you to have anything. So I'm confused there.

Reply:
You're right. Let me re-phrase it.

- retention.bytes
defualt value -> `-1` , which means **disable** the size control for log retention.
supported value row should be removed

Reply:
Ah, I follow now. I've updated. Thanks.